makechrootpkg.in

```diff
@@ -177,13 +177,17 @@ prepare_chroot() {
 
 	$install -d "$copydir"/{build,startdir,{pkg,srcpkg,src,log}dest}
 
-	sed -e '/^MAKEFLAGS=/d' -e '/^PACKAGER=/d' -i "$copydir/etc/makepkg.conf"
-	for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest \
-		"MAKEFLAGS='${MAKEFLAGS:-}'" "PACKAGER='${PACKAGER:-}'"
-	do
-		grep -q "^$x" "$copydir/etc/makepkg.conf" && continue
-		echo "$x" >>"$copydir/etc/makepkg.conf"
-	done
+	[[ -f c"$USER_HOME/.makepkg.conf" ]] && \
```

Comment:
The user `makepkg.conf ` has two location according to `makepkg.conf(5)`, so this check needs to be expanded.

"$XDG_CONFIG_HOME/pacman/makepkg.conf, ~/.makepkg.conf"

Reply:
what does that `c` do there after `-f`?